armbian/base/customize-armbian-rockpro64.sh

```diff
@@ -500,6 +500,22 @@ curl --retry 5 -SLo "/opt/shift/hsm/firmware-bitboxbase.signed.bin" \
   "https://github.com/digitalbitbox/bitbox02-firmware/releases/download/firmware-bitboxbase%2Fv${HSM_VERSION}/firmware-bitboxbase.v${HSM_VERSION}.signed.bin"
 
 
+# AYUFAN KERNEL ---------------------------------------------------------------
+mkdir -p /usr/local/src/ayufan
+cd /usr/local/src/ayufan
+
+AYUFAN_KERNEL_VERSION="4.4.202-1237-rockchip-ayufan"
+AYUFAN_KERNEL_ID="gfd4492386213"
+
+curl --retry 5 -SLO "https://github.com/ayufan-rock64/linux-kernel/releases/download/${AYUFAN_KERNEL_VERSION}/linux-firmware-image-${AYUFAN_KERNEL_VERSION}-${AYUFAN_KERNEL_ID}_${AYUFAN_KERNEL_VERSION}_arm64.deb"
+curl --retry 5 -SLO "https://github.com/ayufan-rock64/linux-kernel/releases/download/${AYUFAN_KERNEL_VERSION}/linux-image-${AYUFAN_KERNEL_VERSION}-${AYUFAN_KERNEL_ID}_${AYUFAN_KERNEL_VERSION}_arm64.deb"
+
+dpkg --install linux-firmware-image-4.4.202-1237-rockchip-ayufan-gfd4492386213_4.4.202-1237-rockchip-ayufan_arm64.deb
```

Comment:
Isn't the name of the file the same as in the curl command? In that case you could use a variable instead of repeating the same string.

Reply:
Good catch, meant to replace that as well. One of the reasons to abstract it into the variables.

Reply:
Fixed with force-pushed commit https://github.com/digitalbitbox/bitbox-base/pull/326/commits/b4ed5f9197944316c8359c11803d3937047383a4